### DISCUSSION TITLE
matlab language support

### DISCUSSION THREAD

**Discussion by felix-mue** [VSCodium/vscodium#1391] [Q&A] [answered]
Is there a way to add language support (coloring etc) for matlab to vscodium?
I find nothing in the package manager, although vs code does have a package for matlab. In the list of languages to choose from, it isn't present as well. I would also take support for the octave language instead. ;)

Cheers!

**Comment by daiyam** [comment, marked as answer]
You should push the extension dev to publish on open-vsx like https://github.com/Gimly/vscode-matlab/issues/166
